tensorflow/python/ops/math_ops.py

```diff
@@ -356,6 +356,26 @@ def _mul(x, y, name=None):
 @tf_export("math.subtract", "subtract")
 @dispatch.add_dispatch_support
 def subtract(x, y, name=None):
+  """Returns 'x' - 'y' element wise
+  
+  For example:
+  >>> x = tf.constant([32])
+  >>> y = tf.constant([34])
+  >>> tf.math.subtract(x, y)
+  <tf.Tensor: shape=(1,), dtype=float32, numpy=array([-2.], dtype=float32)>
+  
+  Args:
+  'x': A 'Tensor'. Must be one of the following types:
```

Comment:
These should be indented. See example at `_neg` below. Please make sure to follow the same style throughout the PR. Including for `Returns`.

Reply:
Not solved yet